app.rb

```diff
@@ -0,0 +1,9 @@
+# app.rb
+require 'sinatra'
+
+
+class ExampleApp < Sinatra::Base
```

Comment:
I think we're all set to use Sinatra (unless @cthelander has any major objections :) ), so I think we can go ahead and name our app class here. `SPiWeb` or something descriptive?